### PR TITLE
Add more fallback urls

### PR DESCRIPTION
**Pull request by Goooler**
Some HTTPS variants.

### CODE REVIEW

File: src/main/kotlin/app/cash/licensee/licenses.kt [hunk 1]
```diff
@@ -54,41 +54,72 @@ private fun PomLicense.toSpdx(): List<SpdxLicense> {
...
       "http://www.apache.org/licenses/LICENSE-2.0.txt",
       "https://www.apache.org/licenses/LICENSE-2.0.txt",
       "http://www.apache.org/licenses/LICENSE-2.0.html",
       "https://www.apache.org/licenses/LICENSE-2.0.html",
       "http://www.opensource.org/licenses/apache2.0.php",
+      "https://www.opensource.org/licenses/apache2.0.php",
       "http://www.apache.org/licenses/LICENSE-2.0",
+      "https://www.apache.org/licenses/LICENSE-2.0",
+      "http://api.github.com/licenses/apache-2.0",
+      "https://api.github.com/licenses/apache-2.0",
```
Comment (Goooler) [inline src/main/kotlin/app/cash/licensee/licenses.kt:68]: I'm not sure if we can use these from https://api.github.com/licenses

Comment (JakeWharton) [reply src/main/kotlin/app/cash/licensee/licenses.kt:68]: Do you have an example project where they are being used?

Comment (Goooler) [reply src/main/kotlin/app/cash/licensee/licenses.kt:68]: https://jitpack.io/com/github/topjohnwu/libsu/service/5.1.0/service-5.1.0.pom

Comment (Goooler) [reply src/main/kotlin/app/cash/licensee/licenses.kt:68]: Seems they are generated by Jitpack automatically if there is no license declared in publication configs.

Comment (JakeWharton) [reply src/main/kotlin/app/cash/licensee/licenses.kt:68]: Another reason to tell people to avoid Jitpack, I guess. I'll still merge it.

**Pull request merged**